cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -125,6 +125,8 @@ protected void createContent(final Bundle savedInstanceState) {
             "--enable-features=LogJsConsoleMessages",
             // Disable rescaling Webpage.
             "--force-device-scale-factor=1",
+            // Enable Cobalt-specific JS attributes/methods.
+            "--enable-blink-features=H5vccAudioConnectors,SourceBufferWriteHead",
```

Comment:
Wait @kaidokert and @yell0wd0g to confirm this is the right way to add new idl attributes in blink.

Reply:
I'll address this in the writeHead PR, but I'll go ahead with a partial interface approach.